### PR TITLE
WRQ-156: [Codecov] Increase coverage for editable scroller

### PR DESCRIPTION
**Pull request by vJIYEv**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Unit Test of editalbe scroller needed to be added.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
I added 2 tests for editable scroller.
- should have selected class and custom selected class when item is selected
- should not have selected class and custom selected class and should have custom focused class when item is unselected

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-156

### Comments
Enact-DCO-1.0-Signed-off-by: Jiye Kim ([jiye.kim@lge.com](mailto:jiye.kim@lge.com))

### CODE REVIEW

**Comment by seunghoh** [comment]
The 2nd commit message does not reflect what you've changed and we don't use the PR title for the commit message
CC: @juwonjeong

**Comment by vJIYEv** [comment]
> The 2nd commit message does not reflect what you've changed and we don't use the PR title for the commit message CC: @juwonjeong

I was confused with 'commit' and 'push' and thought 2nd commit would be pushed as 1st commit because I pushed 2 commits at once.
The commit message is wrong because of this.
And I will not use the PR title for the commit message from now. Thank you.

File: Scroller/tests/EditableWrapper-specs.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	afterEach(() => {
+		items = null;
+		editableScroller = null;
+	});
+
+	describe('Select Item', () => {
+		test(
+			'should have "selected" class when item is selected',
+			() => {
+				render(editableScroller);
```
Comment (juwonjeong) [inline Scroller/tests/EditableWrapper-specs.js:87]:
```suggestion
				render(<Scroller
						direction="horizontal"
						editable={{
							css: {
								'selected': 'customSelected',
								'focused': 'customFocused'
							},
							onComplete: jest.fn(),
							removeItemFuncRef: jest.fn(),
							hideItemFuncRef: jest.fn(),
							showItemFuncRef: jest.fn(),
							focusItemFuncRef: jest.fn(),
							selectItemBy: 'press'
						}}
					>
						{editableScrollerContents}
					</Scroller>);
```

File: Scroller/tests/EditableWrapper-specs.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import '@testing-library/jest-dom';
+import {render, fireEvent, screen} from '@testing-library/react';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
+import Button from '../../Button';
+import IconItem from '../../IconItem';
+import Scroller from '../Scroller';
+import css from '../Scroller.module.less';
```
Comment (juwonjeong) [inline Scroller/tests/EditableWrapper-specs.js:7]: Please remove this line. It is not appropriate to import this less file.

**Comment by juwonjeong** [comment]
I think we can add 2 cases to verify that it has custom selected class and does not have custom focused class when item is selected.

File: Scroller/tests/EditableWrapper-specs.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import '@testing-library/jest-dom';
+import {render, fireEvent, screen} from '@testing-library/react';
+import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator';
```
[the remark this answers is not in the document]
Comment (vJIYEv) [reply Scroller/tests/EditableWrapper-specs.js]: I moved line 3 after import statements of other components.

File: Scroller/tests/EditableWrapper-specs.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+import Button from '../../Button';
```
[the remark this answers is not in the document]
Comment (vJIYEv) [reply Scroller/tests/EditableWrapper-specs.js:4]: I accepted your suggestion.

File: Scroller/tests/EditableWrapper-specs.js (hunk 1 above)
Comment (vJIYEv) [reply Scroller/tests/EditableWrapper-specs.js]: I accepted your suggestion.

File: Scroller/tests/EditableWrapper-specs.js (hunk 2 above)
Comment (vJIYEv) [reply Scroller/tests/EditableWrapper-specs.js]: I accepted your suggestion.

File: Scroller/tests/EditableWrapper-specs.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (juwonjeong) [reply Scroller/tests/EditableWrapper-specs.js:4]: `import SpotlightContainerDecorator from '@enact/spotlight/SpotlightContainerDecorator;` also needs to be sorted again

[the remark this answers is not in the document]
Comment (vJIYEv) [reply Scroller/tests/EditableWrapper-specs.js:5]: I accepted your suggestion.

File: Scroller/tests/EditableWrapper-specs.js [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+const pressEnterKey = keyDownUp(13);
+
+const populateItems = ({index}) => {
+	const iconItemProps = {
+		background: '#ffffff',
+		bordered: true,
+		icon: '',
+		image: null,
+		label: null,
+		labelColor: null,
```
Comment (juwonjeong) [inline Scroller/tests/EditableWrapper-specs.js:23]: I think null is not proper value for labelColor. Could you please check iconItemProps?

Comment (vJIYEv) [reply Scroller/tests/EditableWrapper-specs.js]: I changed iconItemProps values. Please check.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**